### ISSUE TITLE
AWS: Make not found

### ISSUE THREAD

**Issue by MrSerth** [openHPI/poseidon#166]
All users on AWS are experiencing issues with CodeOcean. For them, the execution is no longer working as `make` cannot be found:

```
sh: make: command not found

The last code run exited with a failure (status code 127).
```

As seen in the [AWS exercise](https://codeocean.openhpi.de/exercises/1005), this is bound to the AWS method and probably started on Sunday, June 12th.
@mpass99 Can you have a look, please?

**Comment by MrSerth** [comment]
Temporarily, I disabled the AWS experiment to recover the functionality for all learners. A redeploy of CodeOcean will enable that again.

**Comment by mpass99** [comment]
I assume it is about #147. We added a prefix and so `unset \"${!AWS@}\" && make run` is no longer matching our regex `^make(?:\s+(?<startRule>\w*))?(?<assignments>(?:.*?=.*?)+)?$`. Here is definitely one e2e test missing...

**Comment by MrSerth** [comment]
Ah, that makes sense! But this will be tricky to resolve as well, isn't it?

**Comment by MrSerth** [comment]
Looks good. I was shortly wondering whether we have a bad backtracking behavior (due to the `.*` parts) but that didn't turn out to be true in my short tests. Hence, I would go for it.

**Comment by mpass99** [comment]
It is now deployed to AWS and seems to work. I will add an e2e test later when I'm less in hurry..

**Comment by MrSerth** [comment]
@mpass99 We're experiencing the same error again as of now:

```
sh: make: command not found
```

**Comment by MrSerth** [comment]
> @mpass99 We're experiencing the same error again as of now:
> 
> ```
> sh: make: command not found
> ```

Seems to be fixed again, I am able to execute Java using AWS. (We're still including the AWS env variables, which I expect to be fixed with #168).

**Comment by MrSerth** [comment]
> My mistake. Let's do better next time: #171

Don't worry, thanks for creating the ticket!
